integration/externalbuilders/binary/bin/run

```diff
@@ -14,6 +14,12 @@ fi
 OUTPUT=$1
 ARTIFACTS=$2
 
+# when set, keep track of the artifacts directories to verify
+# proper cleanup in integration test
+if [ -n "${RUN_ARTIFACTS_FILE+1}" ]; then
```

Comment:
curious: What is the `+1` for?

Reply:
This is to avoid an unbound variable for the integration tests that don't need to set this environment variable (aka every other one besides the one I've updated in this PR). Not sure it's the "best" way to handle it but it seemed decent after consulting around the internet. :)